<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Categoria</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lista Categoria</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>271</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Categoria</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar Categoria</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Direccion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Categoria:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nombre">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Direccion">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Telefono">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Path">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_CategoriaCrearContacto">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_ImagenCrearContacto">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>221</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar Imagen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListWidget" name="listWidget_Contacto_Editar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>256</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Contactos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Telefono_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_ImagenCrearContacto_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>221</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Direccion_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Categoria:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nombre_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Path_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Categoria_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Direccion:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>310</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Cambiar Imagen</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Categoria_Editar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Contacto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>Administrador de Actividades, Contactos y Categorias</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCategoria">
    <property name="title">
     <string>Categoria</string>
    </property>
    <addaction name="actionAdministrar_Categoria"/>
   </widget>
   <widget class="QMenu" name="menuContacto">
    <property name="title">
     <string>Contacto</string>
    </property>
    <addaction name="actionCrear_Contacto"/>
    <addaction name="actionModificar_Contacto"/>
    <addaction name="actionGraficar_Lista"/>
   </widget>
   <widget class="QMenu" name="menuActividades">
    <property name="title">
     <string>Actividades</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de...</string>
    </property>
   </widget>
   <addaction name="menuCategoria"/>
   <addaction name="menuContacto"/>
   <addaction name="menuActividades"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdministrar_Categoria">
   <property name="text">
    <string>Administrar Categoria</string>
   </property>
  </action>
  <action name="actionCrear_Contacto">
   <property name="text">
    <string>Crear Contacto</string>
   </property>
  </action>
  <action name="actionGraficar_Lista">
   <property name="text">
    <string>Graficar Lista</string>
   </property>
  </action>
  <action name="actionModificar_Contacto">
   <property name="text">
    <string>Modificar Contacto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
